modulemd/v2/tests/ModulemdTests/modulestream.py

```diff
@@ -1035,8 +1034,16 @@ def test_depends_on_stream(self):
                 stream.build_depends_on_stream(
                     'platform', 'f28'), False)
 
-            self.assertEqual(stream.depends_on_stream('base', 'f30'), False)
-            self.assertEqual(stream.depends_on_stream('base', 'f30'), False)
+            self.assertEqual(
+                stream.depends_on_stream(
+                    'base', ''), False)
```

Comment:
I think you misunderstood the requirement here. We don't need tests for a stream name being the empty string. We need a new requires/buildrequires in the v2 YAML document. Something like `perl` in:

```
  dependencies:
  - buildrequires:
      platform: [ f30 ]
      perl: []
    requires:
      platform: [ -f28 ]
      perl: []
```

This test is ONLY for ModuleStreamV2, since there is no way to represent this in the ModuleStreamV1 YAML.